### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by FionaQY**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -679,7 +679,7 @@ testers are expected to do more *exploratory* testing.
     2. Test case: `add n/ <NAME> p/ <PHONE> e/ <EMAIL> a/ <ADDRESS> j/ <JOB> i/ <INCOME>`<br>
        Expected: A client with the given details is added to the end of the list of clients. Details are shown in the results box and any details that are truncated can be seen by using the `view` command with the client's displayed index.
 
-    3. Test case: `add n/ <NAME> p/ <PHONE> e/ <EMAIL> a/ <ADDRESS> j/ <JOB> i/ <INCOME> t/ <TIER> r/ <REMARK> s/ Status`<br>
+    3. Test case: `add n/ <NAME> p/ <PHONE> e/ <EMAIL> a/ <ADDRESS> j/ <JOB> i/ <INCOME> t/ <TIER> r/ <REMARK> s/ <Status>`<br>
```
Comment (iamdiluxedbutcooler) [inline docs/DeveloperGuide.md:682]: Status should be STATUS?

**Review by iamdiluxedbutcooler — CHANGES_REQUESTED** [review]
LGTM! just a minor change :D

**Review by itsme-zeix — APPROVED** [review]
LGTM

**Review by ZShunRen — APPROVED** [review]
LGTM

**Pull request closed without merging**